fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -1076,6 +1076,10 @@ alter_system_clause ::=
     {:
         RESULT = new AddBackendClause(hostPorts, false);
     :}
+    | KW_MODIFY KW_BE_HOST STRING_LITERAL:oldHostPort KW_TO STRING_LITERAL:newHostPort
```

Comment:
why not use MODIFY BACKEND, that's the same style of semantic with ADD/DROP

Reply:
it's a good idea, I will change it